File: backend-java/src/main/java/mx/codemx/modules/duelos/EstadoDuelo.java
package mx.codemx.modules.duelos;

/** Estados por los que pasa un duelo 1 vs 1. */
public enum EstadoDuelo {
    /** Ambos jugadores recibieron el problema y compiten. */
    EN_CURSO,
    /** Alguien resolvió primero (o hubo abandono). */
    TERMINADO
}

File: backend/Modules/Duelos/EstadoDuelo.cs
namespace CodeMX.Api.Modules.Duelos;

/// <summary>Estados por los que pasa un duelo 1 vs 1.</summary>
public enum EstadoDuelo
{
    EnCurso,    // ambos jugadores recibieron el problema y compiten
    Terminado   // alguien resolvió primero (o hubo abandono)
}
